.github/workflows/release.yaml

```diff
@@ -61,47 +61,29 @@ jobs:
     build_sdk:
         name: Build sdk
         needs: [release, packages_to_build]
-        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@sunodo/sdk') }}
+        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@cartesi/sdk') }}
         uses: ./.github/workflows/sdk.yaml
         secrets: inherit
 
     build_anvil:
         name: Build anvil
         needs: [release, packages_to_build]
-        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@sunodo/anvil') }}
+        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@cartesi/anvil') }}
         uses: ./.github/workflows/anvil.yaml
         secrets: inherit
 
     build_devnet:
         name: Build devnet
         needs: [release, packages_to_build]
-        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@sunodo/devnet') }}
+        if: ${{ needs.release.outputs.published == 'true' && contains(fromJSON(needs.packages_to_build.outputs.packages), '@cartesi/devnet') }}
         uses: ./.github/workflows/devnet.yaml
         secrets: inherit
 
-    build_contracts:
```

Comment:
This removal should be at the https://github.com/sunodo/sunodo/pull/462/commits/bd5c57b29edd3f6782097181a07efad85c333f03 commit.